src/c++/perf_analyzer/infer_data_manager.cc

```diff
@@ -85,111 +185,19 @@ InferDataManager::InitInferDataOutput(
 
 cb::Error
 InferDataManager::UpdateInputs(
-    const int stream_index, const int step_index, InferData& infer_data)
+    const size_t thread_id, const int stream_index, const int step_index,
+    InferData& infer_data)
 {
   // Reset inputs for this inference request
   infer_data.valid_inputs_.clear();
 
   for (const auto& input : infer_data.inputs_) {
-    RETURN_IF_ERROR(input->Reset());
-
-    const auto& model_input = (*(parser_->Inputs()))[input->Name()];
-
-    const uint8_t* data_ptr{nullptr};
-    size_t batch1_bytesize;
-    const int* set_shape_values = nullptr;
-    int set_shape_value_cnt = 0;
-
-    // Number of missing pieces of data for optional inputs
-    int missing_data_cnt = 0;
-
-    for (size_t i = 0; i < batch_size_; ++i) {
```

Comment:
Where'd all this deleted stuff go? Was it a copy-and-paste? Or also refactored? Or changed design? Can you summarize for the logic this loop was doing?

Reply:
Hopefully my previous comments make it clear that it all went into init(). Let me know if you want more clarification